docs/releases/changelog-dev.md

```diff
@@ -27,9 +30,9 @@
 
 ### Deprecations / Removals
 
-- Deleted all the files in `execution` and `experiment` directories (Already obsolete).
-  [#749](https://github.com/qilimanjaro-tech/qililab/pull/749)
-
 ### Documentation
 
 ### Bug fixes
+
+get_parameter for QM did not work due to the lack of the variable bus_alias in self.system_control.get_parameter. The variable has been added to the function and now get parameter does not return a crash.
```

Comment:
I think you put the `-` in the wrong place, no?
```suggestion
- get_parameter for QM did not work due to the lack of the variable `bus_alias in self.system_control.get_parameter`. The variable has been added to the function and now get parameter does not return a crash.
```
(also need to be deleted from the next line!)

Reply:
Addressed!